### PR TITLE
Analytics documentation for 4.3

### PR DESCRIPTION
**Pull request by ryanisgrig**
## Description

* Refactored analytics documentation to several pages
* Displaying all telemetry logger events and properties
* Fixed a property mismatch on LuisTelemetryRecognizer to use `DialogId` (same as Waterfall dialog events)

## Related Issue

#808 

## Checklist
- [X] I have updated related documentation

### CODE REVIEW

File: solutions/analytics/docs/analytics-telemetrylogging.md [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+>Node.js
+```js
+const appInsightsClient = new ApplicationInsightsTelemetryClient(botConfig);
+```
+
+## Telemetry Events
+
+### Initializer
+The Bot Framework SDK provides an Application Inisghts Telemetry Initializer to log important identifiers in all custom events.  
+
```
Comment (darrenj) [inline solutions/analytics/docs/analytics-telemetrylogging.md:103]: Tables aren't rendering correctly?

**Review by darrenj — CHANGES_REQUESTED** [review]

File: solutions/analytics/docs/analytics-telemetrylogging.md (hunk 1 above)
Comment (ryanisgrig) [reply solutions/analytics/docs/analytics-telemetrylogging.md:103]: Is this on your machine? They render correctly on mine.

Comment (darrenj) [reply solutions/analytics/docs/analytics-telemetrylogging.md:103]: Just rendering in the github pr viewer, likely that if it works for you!

**Review by darrenj — APPROVED** [review]

**Review by lzc850612 — APPROVED** [review]

File: solutions/analytics/README.md [hunk 2]
```diff
@@ -38,122 +36,16 @@ Sentiment must be enabled for each application.  To enable sentiment:
...
-|[All Conversations Metrics](#all-conversations-metrics)| Highlights the average number of conversations per unique user and the average duration by day.|
-|[Conversations Drill Down](#conversations-drill-down)| Per conversation, this highlights the dialogs triggered and common utterances.|
-|[Transcript](#transcript)| Highlights a count of interactions, session duration, and the transcript between bot and users per unique conversation.|
-|[Demographics](#demographics)| Demonstrates where users are connecting to your bot and the languages they are using.|
-|[Word Cloud](#word-cloud)| Shows a word cloud and tree map of common uttered phrases.|
-|[Sentiment](#sentiment)| If sentiment analysis is enabled, will show the average sentiment by day.|
-|[QnA Maker Insights](#qna-maker-insights)| Shows insights on user utterances matches with QnA Maker. |
+## Telemetry Logging
+[Telemetry Overview](docs/analytics-telemetrylogging) highlights the provided telemetry for bot and user activities, including [LUIS](https://www.luis.ai/) and [QnA Maker](https://www.qnamaker.ai/) results.  how to configure your bot's telemetry, either through bot configuring or overriding the telemetry client.
```
Comment (gabog) [inline solutions/analytics/README.md]: The telemetry overview link doesn't work, I think is missing the .md extension.

File: solutions/analytics/README.md [hunk 3]
```diff
@@ -38,122 +36,16 @@ Sentiment must be enabled for each application.  To enable sentiment:
...
-
-You will need to replace `<app-insights-application-id>` with your Application Insights Application ID, which can be obtained by:
-1. Log in to the [Azure Portal](https://portal.azure.com/).
-2. Navigate to your Application Insights resource.
-3. Select **API Access**.
-4. Copy the Application ID.
-
-![Appinsightsapplicationid](media/appinsightsapplicationid.png)
+## Application Insights Analytics
+[Applications Insights Analytics](docs/analytics-applicationinsights) demonstrates how to execute common queries for for bot analytics and debugging in Visual Studio.
```
Comment (gabog) [inline solutions/analytics/README.md]: missing ".md" extension in link?

File: solutions/analytics/README.md [hunk 4]
```diff
@@ -38,122 +36,16 @@ Sentiment must be enabled for each application.  To enable sentiment:
...
-![Example Report](media/powerbi-conversationanalytics-transcript.png)
-#### Demographics
-![Example Report](media/powerbi-conversationanalytics-demographics.png)
-#### Word Cloud
-![Example Report](media/powerbi-conversationanalytics-wordcloud.png)
-#### Sentiment
-![Example Report](media/powerbi-conversationanalytics-sentimentanalysis.png)
-#### QnA Maker Insights
-![Example Report](media/powerbi-conversationanalytics-qnamakerinsights.png)
+The [Power BI Analytics sample](docs/analytics-powerbi) shows example Power BI dashboards to gain insights on your bot's performance and quality.
```
Comment (gabog) [inline solutions/analytics/README.md]: missing .md extension in link?

File: solutions/analytics/docs/analytics-telemetrylogging.md [hunk 5]
```diff
@@ -0,0 +1,257 @@
...
+ `user_id` |  [ChannelId](https://github.com/Microsoft/botframework-obi/blob/master/botframework-activity/botframework-activity.md#channel-id) + [From.Id](https://github.com/Microsoft/botframework-obi/blob/master/botframework-activity/botframework-activity.md#from)
+`session_id` |  [ConversationId](https://github.com/Microsoft/botframework-obi/blob/master/botframework-activity/botframework-activity.md#conversation)
+`customDimensions.activityId` | [The bot activity Id](https://github.com/Microsoft/botframework-obi/blob/master/botframework-activity/botframework-activity.md#id)
+`customDimensions.activityType` | [The bot activity type ](https://github.com/Microsoft/botframework-obi/blob/master/botframework-activity/botframework-activity.md#channel-id)
+`customDimensions.channelId` | [The bot activity channel Id ](https://github.com/Microsoft/botframework-obi/blob/master/botframework-activity/botframework-activity.md#channel-id)
+
+### Custom Events
+
+#### BotMessageReceived
+*Logs activites sent from users to the bot*
```
Comment (gabog) [inline solutions/analytics/docs/analytics-telemetrylogging.md:115]: This section doesn't look right when I a look at it in the browser, formatting issue?
image

File: solutions/analytics/docs/analytics-applicationinsights.md [hunk 6]
```diff
@@ -1,21 +1,22 @@
...
-| summarize count() by name
+| where name startswith "LuisResult"
+| extend intent = tostring(customDimensions.['intent'])
+| summarize count() by intent
 | order by count_ desc
 | render piechart
 ```
 
-### P50, P95, P99 for Services
+## P50, P95, P99 for Services
```
Comment (gabog) [inline solutions/analytics/docs/analytics-applicationinsights.md:19]: Note, some VS images are missing at the bottom of this page:
image

**Pull request merged**